sentry-ruby/lib/sentry/log_event.rb

```diff
@@ -0,0 +1,113 @@
+# frozen_string_literal: true
+
+module Sentry
+  # Event type that represents a log entry with its attributes
+  #
+  # @see https://develop.sentry.dev/sdk/telemetry/logs/#log-envelope-item-payload
+  class LogEvent < Event
+    TYPE = "log"
+
+    SERIALIZEABLE_ATTRIBUTES = %i[
+      level
+      body
+      timestamp
+      trace_id
+      attributes
+    ]
+
+    SENTRY_ATTRIBUTES = {
+      "sentry.trace.parent_span_id" => :parent_span_id,
+      "sentry.environment" => :environment,
+      "sentry.release" => :release,
+      "sentry.server_name" => :server_name,
```

Comment:
```suggestion
      "server.address" => :server_name,
```

This follows the OpenTelemetry semantic conventions for server, which we want to follow: https://opentelemetry.io/docs/specs/semconv/attributes-registry/server/#server-attributes